package/db/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = ["setuptools", "wheel"]  # PEP 508 specifications.
```

Comment:
Not sure what you meant here.
The initial issue was about the missing `wheel` dependency on the `steup.py` file for this very package. Without touching the `setup.py` file, the build would fail because it defines a custom wheel class  and thus imports the `wheel` package. when building metadata.

A solution you proposed in the original issue was to simply provide a backup import with try except so that the `setup.py` can be run at the meta_build phase, but it sounded a bit hacky to me.

Another solution would be to either completely rewrite the `setup.py` in order to never import the wheel class, and provide the wanted specific features by pure python code, or to use a whole another build backend altogether

Reply:
Yeah, I think I confused several files you have here. I propose that a comment is added here to state that this entry should be removed as soon as `setup.py` stops importing `wheel`..